pkg/loghttp/push/push_test.go

```diff
@@ -61,12 +61,13 @@ func TestParseRequest(t *testing.T) {
 		contentEncoding                 string
 		valid                           bool
 		enableServiceDiscovery          bool
-		expectedStructuredMetadataBytes int
-		expectedBytes                   int
-		expectedLines                   int
+		expectedStructuredMetadataBytes map[string]int
```

Comment:
I think we should add a test here with multiple named and unnamed policies

Reply:
Also passing nil as the policyResolver, in which case only one item per map should be expected ("")